vm/chef/cookbooks/resourcespace/files/resourcespace

```diff
@@ -19,6 +19,7 @@ source /opt/c2d/resourcespace-utils || exit 1
 readonly app_name="$(get_attribute_value "app_name")"
 
 readonly mysql_db="resourcespace"
+readonly mysql_root_password="$(get_attribute_value "mysql-root-password")"
```

Comment:
might be good to make this "mysql_root_password" to match the lower snake case naming convention of other metadata keys.